README.md

```diff
@@ -12,6 +12,65 @@ _A collection of Android libraries to build browsers or browser-like application
+    │                     │ │ └ ─ ─ ─ ─ ─ ─ ┘ └ ─ ─ ─ ─ ─ ─ ┘  └ ─ ─ ─ ─ ─ ─ ┘  └ ─ ─ ─ ─ ─ ─ ┘ │ │
+    │                     │ └───────────────────────────────────────────────────────────────────┘ │
+    ├─────────────────────┼───────────────────────────────────────────────────────────────────────┤
+    │                     │ ┌───────────────────────────────────────────────────────────────────┐ │
+    │  Abstractions and   │ │                              Concept                              │ │
+    │   contracts for     │ │ ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─    ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─   ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─  │ │
+    │     component       │ │        Engine      │     Session Storage  │        Toolbar      │ │ │
+    │  implementations    │ │ └ ─ ─ ─ ─ ─ ─ ─ ─ ─    └ ─ ─ ─ ─ ─ ─ ─ ─ ─   └ ─ ─ ─ ─ ─ ─ ─ ─ ─  │ │
+    │                     │ └───────────────────────────────────────────────────────────────────┘ │
+    ├─────────────────────├───────────────────────────────────────────────────────────────────────┤
+    │                     │ ┌────────────────────────────────┐ ┌────────────────────────────────┐ │
+    │                     │ │               UI               │ │            Service             │ │
+    │     Standalone      │ │    ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │    ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │
+    │     components      │ │          Autocomplete     │    │ │           Telemetry       │    │ │
+    │                     │ │    └ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │    └ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │
+    │                     │ │    ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ └────────────────────────────────┘ │
+    │                     │ │            Progress       │    │ ┌────────────────────────────────┐ │
+    │                     │ │    └ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │            Support             │ │
+    │                     │ │    ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │    ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │
+    │                     │ │             Colors        │    │ │       Kotlin extensions   │    │ │
+    │                     │ │    └ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │    └ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │
+    │                     │ │    ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │    ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │
+    │                     │ │             Fonts         │    │ │           Utilities       │    │ │
+    │                     │ │    └ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │    └ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │
+    │                     │ │    ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │                                │ │
+    │                     │ │             Icons         │    │ │                                │ │
+    │                     │ │    └ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─     │ │                                │ │
+    │                     │ └────────────────────────────────┘ └────────────────────────────────┘ │
+    └─────────────────────┴───────────────────────────────────────────────────────────────────────┘
```

Comment:
This looks pretty cool. But I fear that this will be outdated pretty fast and a pain to update. :)

Now that we are not using the wiki anymore we can also put images into docs/images and integrate them.